apps/src/lib/levelbuilder/lesson-editor/FindVocabularyDialog.jsx

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import DialogFooter from '@cdo/apps/templates/teacherDashboard/DialogFooter';
+import Button from '@cdo/apps/templates/Button';
+import {connect} from 'react-redux';
+import {vocabularyShape} from '@cdo/apps/lib/levelbuilder/shapes';
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20,
+    fontFamily: '"Gotham 4r", sans-serif, sans-serif'
+  }
+};
+
+class FindVocabularyDialog extends Component {
+  static propTypes = {
+    isOpen: PropTypes.bool.isRequired,
+    handleConfirm: PropTypes.func.isRequired,
+    handleClose: PropTypes.func.isRequired,
+    vocabularies: PropTypes.arrayOf(vocabularyShape)
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedVocabularyKey:
+        this.props.vocabularies.length > 0 ? this.props.vocabularies[0].key : ''
```

Comment:
Is there are no vocabularies should we just disable the button instead?

Reply:
Probably also should consider this for resources but that doesn't have to happen here.

Reply:
I like this -- some courses may not even use Vocab (or Resources), so this sounds like a great way to reduce clutter.

Reply:
I like that! I'll do that, at least for vocabulary in this PR.

Reply:
Done for vocab -- I'll do this for resources in a follow-up PR